runtimes/pythonrt/pythonrt_test.go

```diff
@@ -141,3 +143,47 @@ func Test_pySvc_Run(t *testing.T) {
 	_, err = run.Call(ctx, fn, nil, kwargs)
 	require.NoError(t, err, "call")
 }
+
+var isGoodVersionCasess = []struct {
+	version Version
+	ok      bool
+}{
+	{Version{2, 6}, false},
+	{Version{3, minPyVersion.Minor - 1}, false},
+	{Version{3, minPyVersion.Minor}, true},
+	{Version{3, minPyVersion.Minor + 1}, true},
+}
+
+func Test_isGoodVersion(t *testing.T) {
+	for _, tc := range isGoodVersionCasess {
```

Comment:
- it's the same file/same dir in which you are adding versions. Consider to create it once + chmod before for loop?
- how this file is deleted?